R/deployApp.R

```diff
@@ -373,14 +386,15 @@ deploymentTarget <- function(appPath, appName, account, server = NULL) {
     }
     accountDetails <- accountInfo(account, server)
     createDeploymentTarget(
-      tools::file_path_sans_ext(basename(appPath)),
-      account, accountDetails$server)
+      generateAppName(appTitle, appPath, account),
```

Comment:
Why are we unconditionally generating the app name here? Are we unable to use the one that was conditionally generated previously? (ie. only generating name when necessary)